Presentation/UINavigationController+Presenting.swift

```diff
@@ -78,6 +79,7 @@ public extension UINavigationController {
 
 private class PushPoper: NSObject {
     let vc: UIViewController
+    let vcPrefersNavBarHidden: Bool?
     let animated: Bool
     let disableCoalecing: Bool
```

Comment:
Instead of store vcPrefersNavBarHidden and disableCoalecing we could probably just store options instead? (this does not apply to animated though)

Reply:
Yes. `animated` is not considered directly here. I was also tempted to pass on the options directly to PushPoper. But I see there are many options that have nothing to do with PushPopers. Holding them inside PushPopers seemed unnecessary to me so I decided to keep PushPoper independent of those. Do we have any strong reason to hold all the options in PushPoper?